pkg/slo-controller/noderesource/noderesource.go

```diff
@@ -195,6 +196,30 @@ func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device
 	return err
 }
 
+func (r *NodeResourceReconciler) updateGPUDriverAndModel(node *corev1.Node, device *schedulingv1alpha1.Device) error {
+	if device == nil || device.Labels == nil {
+		return nil
+	}
+
+	if device.Labels[extension.GPUModel] == "" && device.Labels[extension.GPUDriver] == "" {
+		return nil
+	}
+
+	copyNode := node.DeepCopy()
+	if copyNode.Labels == nil {
+		copyNode.Labels = make(map[string]string)
+	}
+	copyNode.Labels[extension.GPUModel] = device.Labels[extension.GPUModel]
+	copyNode.Labels[extension.GPUDriver] = device.Labels[extension.GPUDriver]
+
+	patch := client.MergeFrom(node)
+	if err := r.Client.Patch(context.Background(), copyNode, patch); err != nil {
+		klog.Errorf("patch node gpu model and version fail, err:%v", err)
```

Comment:
maybe

failed to patch node gpu model and version, err:%v

is better?

Reply:
fixed